peer/peer.go

```diff
@@ -803,7 +808,9 @@ func (p *Peer) localVersionMsg() (*wire.MsgVersion, error) {
 
 	// Version message.
 	msg := wire.NewMsgVersion(ourNA, theirNA, nonce, blockNum)
-	msg.AddUserAgent(p.cfg.UserAgentName, p.cfg.UserAgentVersion)
+	msg.AddUserAgent(p.cfg.UserAgentName,
+		p.cfg.UserAgentVersion,
```

Comment:
Please keep this on the original line and add the new parameter on this new line so it is consistent with the rest of the code base.

```Go
msg.AddUserAgent(p.cfg.UserAgentName, p.cfg.UserAgentVersion,
	p.cfg.UserAgentComments...)
```